src/common/io_exerciser/IoOp.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
+#include <string>
+#include <memory>
+#include "include/ceph_assert.h"
+
+/* Overview
+ *
+ * enum OpType
+ *   Enumeration of different types of I/O operation
+ *
+ * class IoOp
+ *   Stores details for an I/O operation. Generated by IoSequences
+ *   and applied by IoExerciser's
+ */
+
+namespace ceph {
+  namespace io_exerciser {
+
+    enum OpType {
```

Comment:
If possible - pls use class enums for all new code.

Reply:
Done